src/app/theme/panel.css

```diff
@@ -35,11 +35,11 @@
 }
 
 .ck.ck-balloon-panel [aria-label="Mermaid toolbar"] .ck-button {
-	line-height: 1;
+	line-height: 21px;
 }
 
 .ck.ck-balloon-panel [aria-label="Mermaid toolbar"] .ck.ck-toolbar__separator {
-	margin-right: var(--ck-spacing-medium);
+	margin-right: var(--ck-spacing-small);
```

Comment:
I think that with left/right margin set to 0 it looks better:

![](https://user-images.githubusercontent.com/5353898/157664168-1d0a7904-919a-4950-895d-ce019bfe7407.png)

vs

![image](https://user-images.githubusercontent.com/5353898/157664274-7f58e01f-9486-494d-ae7b-5fd487c9353d.png)